collector/nginx_plus.go

```diff
@@ -100,6 +101,15 @@ func NewNginxPlusCollector(nginxClient *plusclient.NginxClient, namespace string
 			"health_checks_fails":     newStreamUpstreamServerMetric(namespace, "health_checks_fails", "Failed health checks"),
 			"health_checks_unhealthy": newStreamUpstreamServerMetric(namespace, "health_checks_unhealthy", "How many times the server became unhealthy (state 'unhealthy')"),
 		},
+		streamZoneSyncMetrics: map[string]*prometheus.Desc{
+			"bytes_in":        newStreamZoneSyncMetric(namespace, "bytes_in", "Bytes received by this zone"),
```

Comment:
in the description of bytes_in, bytes_out, msgs_in, msgs_out, we need to use "node" instead of "zone"